docs/orms.rst

```diff
@@ -322,6 +322,22 @@ To work, this class needs an `SQLAlchemy`_ session object affected to the :attr:
     .. attribute:: force_flush
 
         Force a session flush() at the end of :func:`~factory.alchemy.SQLAlchemyModelFactory._create()`.
+        
+        This option is deprecated. Use ``sqlalchemy_session_persistence`` instead.
```

Comment:
Can you style this with `.. note::` so that it stands out that it's deprecated... ?